content/working_with/console/widgets/deploymentsView.md

```diff
@@ -283,6 +283,9 @@ The _Run Workflow_ bulk action will show a modal that allows selecting a
 workflow that will be executed on each deployment that matches the current
 filter.
 
```

Comment:
Optional: 

Maybe we should also say something about the workflows listing - that only workflows common for all deployments matching the current filter are allowed to use? 🤔

Reply:
Sure, seems like a nice idea 👍

Reply:
Adjusted within 6963aaf51124c3bd03f773a8eb4e1a9bbb1c31a6